spec/string_foundation/length_spec.rb

```diff
@@ -0,0 +1,36 @@
+# ==============================================================================
+# SPEC - STRING FOUNDATION - LENGTH
+# ==============================================================================
+# frozen_string_literal: true
+describe "[ Length Methods ]" do
+
+  describe "COMPARE CHARACTERS LENGTH" do
+    context "when the argument is String class," do
+      it { expect{ "string".length?("6") }.to raise_error ArgumentError }
```

Comment:
When you use `subject` and `is_expected` methods, you can write with oneline. Because it will be wrong English.

```rb
# NG
# (Wrong English: It expect XXX to equal YYY.)
it { expect(xxx).to eq(yyy) }

# OK
# (Right English: It is expected to equal YYY.)
subject { xxx }
it { is_expected.to eq yyy }
```

Reply:
In this method, it is possible to write simply using `subject` and `let` method.

```rb
let(:string) { "" }
let(:expected_length) { 0 }
subject { string.legth?(expected_length) }

context "..." do
  let(:string) { "abc" }
  let(:expected_length) { 3 }

  it { is_expected.to be true }
end

context "..." do
  let(:string) { "abcd" }
  let(:expected_length) { 3 }

  it { is_expected.to be false }
end
```